### DISCUSSION TITLE
Week 2 (2023 KW 42, 16.10.23-22.10.23) - Collecting requirements

### DISCUSSION THREAD

**Discussion by benedictweis** [SE-TINF22B2/G5-DuoGradus#18] [Statusberichte]
### ✅ Was diese Woche passiert ist?

- Rollen im Projekt abgesprochen
- Aufgaben neu angeordnet (durch neue Erkenntnisse in der Vorlesung)
- Sammeln von Anforderungen als Github Discussions und nach in der Vorlesung besprochenem Prinzip #13 
- Kategorisieren der Anforderungen in `required` und `optional` #17
- Priorität der Anforderungen feststellen durch "voting" auf die Anforderungen (Github Voting feature bei Discussions)

### 👣 Was sind die nächsten Schritte?

- Anforderungen nochmals als Team durchgehen und zeitlich sortieren sowie in Teilschritte Zerlegen
- Eventuell entstehen dabei neue Anforderungen oder fallen weg
- (im Laufe der Vorlesung ergaben sich noch einige mehr) 

### 🤺 Was waren eventuelle Hindernisse?

- Soweit keine 🎉

### 💡 Was waren die learnings dieser Woche?

- Projektmanagement ist sehr komplex 😔
- Gemeinsam Anforderungen zu bestimmen hat zu vielen neuen Ideen geführt auf die kein einzelner von uns hätte kommen können

**Comment by henrybrink** [comment]
"Priorität" nicht "Wichtigkeit"
